heft-plugins/heft-storybook-plugin/src/StorybookRunner.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { Terminal } from '@rushstack/node-core-library';
+
+import { IScopedLogger } from '@rushstack/heft';
+import {
+  ISubprocessRunnerBaseConfiguration,
+  SubprocessRunnerBase
+} from '@rushstack/heft/lib/utilities/subprocess/SubprocessRunnerBase';
```

Comment:
Should we consider moving this to node-core-library and genericizing it?

Reply:
In the PR description I suggested making it a Heft API. Maybe it is more general than that?

Outside the scope of this PR, however.